### PR TITLE
Added Desktop Shortcut feature - usage:

### PR DESCRIPTION
**Pull request by sidewinder040**
### Desktop Shortcut feature
Hi Martin,
`quickemu --shortcut --vm <VMNAME>.conf`
.desktop added to user profile `~/.local/share/applications folder`
The icon for virgil is used.
When the VM is --deleted, the .desktop file is removed.
Updated the README.md also.

 TODO Proposal - Add parameters options to be added to <vm>.conf file:

- SHORTCUT=1 # Add shortcut when vm is created
- ICON=<path to custom icon> # User can add distro logo icon from web

This is my first Git Pull request, i hope I've submitted it correctly.

Regards, 
Mark Crouch

### CODE REVIEW

**Comment by sidewinder040** [comment]
There's some debug output needs deleting:
```
shortcut is 1
VM= focal-desktop-daily.conf
```
Can you remove these lines - thanks.

File: quickemu [hunk 1]
```diff
@@ -441,6 +465,10 @@ while [ $# -gt 0 ]; do
       VM="${2}"
       shift
       shift;;
+    -shortcut|--shortcut)
+      SHORTCUT=1
+      echo "shortcut is" ${SHORTCUT}
```
Comment (sidewinder040) [inline quickemu]: Debug Code - Please delete

File: quickemu [hunk 2]
```diff
@@ -377,13 +383,30 @@ function vm_boot() {
    fi
 }
 
+function shortcut_create {
+  local VMNAME=$(basename "${VM}" .conf)
+  local LAUNCHER_DIR="$(dirname "$(realpath "$0")")"
+  local filename="/home/${USER}/.local/share/applications/${VMNAME}.desktop"
```
Comment (sidewinder040) [inline quickemu]: Please delete line - variable is not used.

**Comment by AnErrupTion** [comment]
Seems a nice thing, gotta add this into my fork lel

File: quickemu [hunk 3]
```diff
@@ -377,13 +383,30 @@ function vm_boot() {
...
+  cat << EOF > ${filename}
+[Desktop Entry]
+Version=1.0
+Type=Application
+Terminal=true
+Exec=${LAUNCHER_DIR}/${LAUNCHER} --vm ${VM}
+Name=${VMNAME}
+Icon=/snap/qemu-virgil/74/meta/gui/icon.png
```
Comment (sidewinder040) [inline quickemu]: Icon=/snap/qemu-virgil/74/meta/gui/icon.png
Please change to:
Icon=/snap/qemu-virgil/**current**/meta/gui/icon.png

**Comment by sidewinder040** [comment]
I've had time to process the changes needed to tidy up, so I'll resubmit this feature pull request

**Pull request closed without merging**